tamr_unify_client/client.py

```diff
@@ -23,6 +23,18 @@ def successful(self):
 Response.successful = successful
 
 
+def fix_base_path(base_path):
+    if not base_path:
```

Comment:
This is an undocumented change to the contract of how client handles `base_path`, and is not part of this bug.
This change needs to be justified, tested, and documented, or removed.